openmetadata-ui/src/main/resources/ui/cypress/common/common.js

```diff
@@ -13,9 +13,12 @@
 
 /// <reference types="cypress" />
 
+import { SEARCH_INDEX } from '../constants/constants';
+
 export const descriptionBox =
   '.toastui-editor-md-container > .toastui-editor > .ProseMirror';
 export const uuid = () => Cypress._.random(0, 1e6);
+const RETRY_TIMES = 30;
```

Comment:
Can we move this to the constants file?

Reply:
constants file contains the data which we use across all specs, whereas `RETRY_TIMES` is related only to a particular function which is in `common.js` file